Sources/ProjectDescription/TargetDependency.swift

```diff
@@ -60,6 +60,13 @@ public enum TargetDependency: Codable, Equatable {
     /// - Parameters:
```

Comment:
@tuist/core is there anything I should be addressing as nearly all of the comments have been addresses, have I missed anything?